docs/dev/plugins/cloudformation/cloudformation-basics.md

```diff
@@ -0,0 +1,37 @@
+# AWS CloudFormation
+How to transform a TOSCA model to an Amazon Web Services(AWS) CloudFormation template.
+
+## Mapping
+Mappings of TOSCA node types to AWS Resources are shown in [this table](mapping.md).
+
+## Transformation Lyfecycle
+
+### validate
+- Nothing to do here so far
+
+### prepare
+- Add missing nodes to the graph to fulfill the requirements.
+- Split the graph, each part should representing a AWS Resource (multiple nodes can be put together to a single AWS Resource)
+
+### transform
+- Transform the graph/the combined nodes into a CloudFormation template
+
+### cleanup
+- Nothing to do here so far
+
+## Deployment / Readme
+- The target artifact should be a single template file (YAML)
+- Deployment is handled by using the AWS Command Line Interface decided in [this adr](../../../adr/0007-plugin-cloudformation-cli.md)
+
+## Next steps
```

Comment:
Please also consider opening issues instead of writing it into some document.

Issues + A corresponding Epic are way easier to manage.